src/Symfony/Component/Console/Helper/Table.php

```diff
@@ -586,7 +584,7 @@ private function buildTableRows(array $rows): TableRows
                 }
                 $escaped = implode("\n", array_map([OutputFormatter::class, 'escapeTrailingBackslash'], explode("\n", $cell)));
                 $cell = $cell instanceof TableCell ? new TableCell($escaped, ['colspan' => $cell->getColspan()]) : $escaped;
-                $lines = explode("\n", str_replace("\n", "<fg=default;bg=default>\n</>", $cell));
+                $lines = explode("\n", str_replace("\n", "<fg=default;bg=default></>\n", $cell));
```

Comment:
This is the fix, right?

Reply:
yes, because the newline previously split the reset style tag, you get a situation where multi-line output ends up with the closing tag present at the end and the closing tag at the beginning

Reply:
> Please revert all unrelated CS changes, so that the PR is focused on one topic.
The only CS related changes I've made are ones provided by your test/build process, it won't pass without applying them.

Reply:
that's just a tool, it's not bullet proof :)

Reply:
ok, @nicolas-grekas I've removed the extra CS changes